src/plugins/chart_expressions/expression_partition_vis/public/components/partition_vis_component.tsx

```diff
@@ -399,7 +400,7 @@ const PartitionVisComponent = (props: PartitionVisComponentProps) => {
                 showLegendExtra={visParams.showValuesInLegend}
                 onElementClick={(args) => {
                   handleSliceClick(
-                    args[0][0] as LayerValue[],
+                    (args[0] as PartitionElementEvent)[0],
```

Comment:
This is minor, but could we maybe create a variable and name it in a way that it's clear what this value means? I guess it makes this more clear. wdyt?

Reply:
Sure, why not, I was just limiting the changes to the minimum but I'm totally fine to improve that